vendor/wgpu/sdl3glue/glue_windows.odin

```diff
@@ -0,0 +1,23 @@
+package wgpu_sdl3_glue
+
+import win "core:sys/windows"
+
+import  "vendor:sdl3"
+import  "vendor:wgpu"
+
+GetSurface :: proc(instance: wgpu.Instance, window: ^sdl3.Window) -> wgpu.Surface {
+	hwnd := sdl3.GetPointerProperty(sdl3.GetWindowProperties(window), sdl3.PROP_WINDOW_WIN32_HWND_POINTER, nil)
+	hinstance := win.GetModuleHandleW(nil)

```

Comment:
Using `sdl3.PROP_WINDOW_WIN32_INSTANCE_POINTER` is probably more proper than retrieving the main instance through `GetModuleHandleW` here.